python/tests/unit/arcticdb/version_store/test_aggregation.py

```diff
@@ -291,18 +333,47 @@ def test_first_agg_with_append(local_object_version_store):
     assert_frame_equal(vit.data, df)
 
 
-@use_of_function_scoped_fixtures_in_hypothesis_checked
-@settings(deadline=None)
-@given(
-    df=data_frames(
-        [
-            column("grouping_column", elements=string_strategy, fill=string_strategy),
-            column("a", elements=numeric_type_strategies()),
-        ],
-        index=range_indexes(),
-    )
-)
-def test_hypothesis_last_agg_numeric(lmdb_version_store, df):
+def test_first_agg_strings_with_append_local(local_object_version_store):
```

Comment:
This test is segfaulting while the same using `lmdb_version_store` below is working fine.